bolt-sidecar/src/builder/mod.rs

```diff
@@ -117,13 +115,20 @@ impl LocalBuilder {
         let payload_and_blobs = PayloadAndBlobs {
             execution_payload: eth_payload,
             // TODO: add included blobs here
-            blobs_bundle: None,
+            blobs_bundle: Default::default(),
```

Comment:
we should add an issue for this: support blobs across the self building pipeline

Reply:
#105